src/components/PostTable/PostTable.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import config, { timezoneDifference } from '../../config';
+import Header from './Header';
+import {
+  Wrapper, Title, Row, Cell, Link,
+} from './PostTable.styles';
+
+const { linkPrefix } = config.postTable;
+
+function PostTable({ activeCell, data }) {
+  const { day, hour } = activeCell;
+
+  const timeToMiliseconds = 1000;
+  const timeToUserTimezone = (utcTime) => (
+    new Date((utcTime + timezoneDifference) * timeToMiliseconds).getMinutes()
+  );
+  const compareCreationTime = (a, b) => (
+    timeToUserTimezone(a.created_utc) - timeToUserTimezone(b.created_utc)
+  );
+
+  const posts = (day > -1 && day !== null)
+    ? [...data[day][hour]].sort(compareCreationTime)
+    : false;
```

Comment:
Returning `false` is a bit problematic. I would try to always use one type for a variable. Here you have two: array and boolean. The second problem is that you use `posts.map` below. So when `posts` is `false` this would break